src/pages/_document.js

```diff
@@ -0,0 +1,29 @@
+import Document from "next/document";
```

Comment:
The `src/pages/_document.js` file is necessary to inject styles in the page for `styled-components`. Found this solution in Next.js doc.